packages/oas-utils/src/spec-getters/getExampleFromSchema.ts

```diff
@@ -132,9 +132,12 @@ export const getExampleFromSchema = (
     }
   }
 
-  // Use the first example, if there’s an array
+  // Use the first example, if there’s an array, and handle objects
   if (Array.isArray(schema.examples) && schema.examples.length > 0) {
-    return cache(schema, schema.examples[0])
+    const firstExample = schema.examples[0]
+    return typeof firstExample === 'object'
+      ? cache(schema, Object.values(firstExample))
```

Comment:
what if the example is supposed to be an object? Wouldn't we want the whole object?

Reply:
good call i guess it should remain, this was a first quick shot but i'll dig deeper regarding the casting